src/recipes.jl

```diff
@@ -1663,10 +1663,20 @@ end
 
 @userplot AreaPlot
 
-@recipe function f(a::AreaPlot)
+@recipe function f(a::AreaPlot; steptype = :none)
     data = cumsum(a.args[end], dims = 2)
     x = length(a.args) == 1 ? (axes(data, 1)) : a.args[1]
-    seriestype := :line
```

Comment:
why limit it to those seriestypes?

What about 
```julia
seriestype --> :line
```
Then you can pass any seriestype at the call site